File: server/zlj2019/game_server/src/com/imop/lj/gameserver/arena/msg/CGArenaKillCd.java
package com.imop.lj.gameserver.arena.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.arena.handler.ArenaHandlerFactory;

/**
 * 消除竞技场cd
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGArenaKillCd extends CGMessage{
	
	
	public CGArenaKillCd (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_ARENA_KILL_CD;
	}
	
	@Override
	public String getTypeName() {
		return "CG_ARENA_KILL_CD";
	}


	@Override
	public void execute() {
		ArenaHandlerFactory.getHandler().handleArenaKillCd(this.getSession().getPlayer(), this);
	}
}

File: u3u2/Assets/Scripts/app/net/c2s/CGArenaKillCd.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 消除竞技场cd
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGArenaKillCd :BaseMessage
{
	
	
	public CGArenaKillCd ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_ARENA_KILL_CD;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}